hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1912,6 +1871,10 @@ public Option<HoodieLogBlock.HoodieLogBlockType> getLogDataBlockFormat() {
         .map(HoodieLogBlock.HoodieLogBlockType::fromId);
   }
 
+  public void testEnums() {
```

Comment:
need to remove

Reply:
Can we add a test to `TestConfigProperty` which does a string comparison for documentation generated?